src/__tests__/proofValue.json

```diff
@@ -6,6 +6,6 @@
   "merkleRoot": "3c9ee831b8705f2fbe09f8b3a92247eed88cdc90418c024924be668fdc92e781", 
   "targetHash": "c65c6184e3d5a945ddb5437e93ea312411fd33aa1def22b0746d6ecd4aa30f20",
   "anchors": [
-    "blink:mtc:mumbai:582733d7cef8035d87cecc9ebbe13b3a2f6cc52583fbcd2b9709f20a6b8b56b3"
+    "blink:btc:testnet:582733d7cef8035d87cecc9ebbe13b3a2f6cc52583fbcd2b9709f20a6b8b56b3"
```

Comment:
wait why the change on this one? Matic Mumbai should be a valid test case

Reply:
The decoded / encoded value (resultBase58) in tests was not updated after changing anchor in proofValue file to "mtb:mumbai", so the existing tests were failing, here's the commit: https://github.com/vaultie/lds-merkle-proof-2019/commit/6b20737b67e8a136fff3537963fa01de04b48fd4

I can revert that change, and update the resultBase58 as well in the test files, if you'd prefer that.

Reply:
I think the change is ok